controllers/controllers/workloads/env/vcap_app_builder_test.go

```diff
@@ -63,5 +81,70 @@ var _ = Describe("VCAP_APPLICATION env value builder", func() {
 				Expect(vcapAppValue).To(HaveKeyWithValue("x", HaveKeyWithValue("y", "z")))
 			})
 		})
+
+		When("application routes are provided", func() {
+			var (
+				appDomain *korifiv1alpha1.CFDomain
+				appRoute  *korifiv1alpha1.CFRoute
+				routeUri  string
+			)
+
+			BeforeEach(func() {
+				appDomain = &korifiv1alpha1.CFDomain{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: cfApp.Namespace,
+						Name:      "domain-123-456",
+					},
+					Spec: korifiv1alpha1.CFDomainSpec{
+						Name: "mydomain.platform.com",
+					},
+				}
+				ensureCreate(appDomain)
+
+				appRoute = &korifiv1alpha1.CFRoute{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: cfApp.Namespace,
+						Name:      "vcap-application-my-route",
+					},
+					Spec: korifiv1alpha1.CFRouteSpec{
+						DomainRef: v1.ObjectReference{
+							Name: appDomain.Name,
+						},
+						Destinations: []korifiv1alpha1.Destination{korifiv1alpha1.Destination{
+							GUID: "destination-123-456",
+							AppRef: v1.LocalObjectReference{
+								Name: cfApp.Name,
+							},
+						}},
+					},
+				}
+
+				ensureCreate(appRoute)
+
+				routeUri = cfApp.Name + ".mydomain.platform.com"
+
+				ensurePatch(appRoute, func(appRoute *korifiv1alpha1.CFRoute) {
+					appRoute.Status.CurrentStatus = "valid"
+					appRoute.Status.Description = "test patch status"
+					appRoute.Status.URI = routeUri
+				})
+
+				builder = env.NewVCAPApplicationEnvValueBuilder(controllersClient, nil)
+			})
+
+			JustBeforeEach(func() {
+				vcapApplication, buildVCAPApplicationEnvValueErr = builder.BuildEnvValue(ctx, cfApp)
+			})
+
+			It("includes application uris", func() {
+				Expect(buildVCAPApplicationEnvValueErr).ToNot(HaveOccurred())
+				Expect(vcapApplication).To(HaveKey("VCAP_APPLICATION"))
+				vcapAppValue := map[string]any{}
+				Expect(json.Unmarshal([]byte(vcapApplication["VCAP_APPLICATION"]), &vcapAppValue)).To(Succeed())
+				Expect(vcapAppValue).To(HaveKeyWithValue("application_id", cfApp.Name))
+				Expect(vcapAppValue).To(HaveKeyWithValue("application_uris", []any{routeUri}))
```

Comment:
Gomega offers an idiomatic matcher to assert that an array only consists of given elements - `ConsistsOf`

Therefore this line could say 

```
Expect(vcapAppValue).To(HaveKeyWithValue("application_uris", ConsistOf(routeUri)))
```

Same goes for the next `Expect` as well.